pkg/common/events/events.go

```diff
@@ -33,20 +33,20 @@ type SchedulingEvent interface {
 type ApplicationEventType string
 
 const (
-	SubmitApplication    ApplicationEventType = "SubmitApplication"
-	RecoverApplication   ApplicationEventType = "RecoverApplication"
-	AcceptApplication    ApplicationEventType = "AcceptApplication"
-	TryReserve           ApplicationEventType = "TryReserve"
-	UpdateReservation    ApplicationEventType = "UpdateReservation"
-	RunApplication       ApplicationEventType = "RunApplication"
-	RejectApplication    ApplicationEventType = "RejectApplication"
-	CompleteApplication  ApplicationEventType = "CompleteApplication"
-	FailApplication      ApplicationEventType = "FailApplication"
-	KillApplication      ApplicationEventType = "KillApplication"
-	KilledApplication    ApplicationEventType = "KilledApplication"
-	ReleaseAppAllocation ApplicationEventType = "ReleaseAppAllocation"
+	SubmitApplication       ApplicationEventType = "SubmitApplication"
+	RecoverApplication      ApplicationEventType = "RecoverApplication"
+	AcceptApplication       ApplicationEventType = "AcceptApplication"
+	TryReserve              ApplicationEventType = "TryReserve"
+	UpdateReservation       ApplicationEventType = "UpdateReservation"
+	RunApplication          ApplicationEventType = "RunApplication"
+	RejectApplication       ApplicationEventType = "RejectApplication"
+	CompleteApplication     ApplicationEventType = "CompleteApplication"
+	FailApplication         ApplicationEventType = "FailApplication"
+	KillApplication         ApplicationEventType = "KillApplication"
+	KilledApplication       ApplicationEventType = "KilledApplication"
+	ReleaseAppAllocation    ApplicationEventType = "ReleaseAppAllocation"
 	ReleaseAppAllocationAsk ApplicationEventType = "ReleaseAppAllocationAsk"
-	AppStateChange       ApplicationEventType = "ApplicationStateChange"
+	AppStateChange          ApplicationEventType = "ApplicationStateChange"
```

Comment:
do we have any change here other than formating?
if not, prefer not to change this in this PR

Reply:
Ok. I'll roll back this change. It was introduced when I ran `go fmt`